JavaScript/JavaScriptSDK/Util.ts

```diff
@@ -80,6 +80,18 @@
         }
 
         /**
+        * Checks if error has no meaningful data inside. Ususally such errors are received by window.onerror when error
+        * happens in a script from other domain (cross origin, CORS).
+        */
+        public static isMeaninglessError(message: string, url: string, lineNumber: number, columnNumber: number, error: Error): boolean {
```

Comment:
I think this should be configurable. Although the browser strips most information out of this, the fact that a script on the site is throwing errors could still be useful information.